Keda.Cosmosdb.Scaler/src/Services/CosmosDBTriggerMetadata.cs

```diff
@@ -0,0 +1,10 @@
+﻿namespace Keda.CosmosDB.Scaler.Services
+{
+    public class CosmosDBTriggerMetadata
+    {
+        public static string CollectionName { get { return "collectionName"; } }
+        public static string DatabaseName { get { return "databaseName"; } }

```

Comment:
same here. Move to constants class

Reply:
Thinks we can remove this whole class in favor of the proposed constants here